src/test/java/com/taf/tests/steps/CucumberHooks.java

```diff
@@ -0,0 +1,21 @@
+package com.taf.tests.steps;
+
+import com.codeborne.selenide.Configuration;
+import com.codeborne.selenide.Selenide;
+import com.taf.core.ConfigManager;
+import io.cucumber.java.After;
+import io.cucumber.java.BeforeAll;
+
+public class CucumberHooks {
+
+    @BeforeAll
+    public static void setUp() {
+        Configuration.browser = ConfigManager.getBrowser();
+        Configuration.baseUrl = ConfigManager.getUrl();
+    }
```

Comment:
as a suggestion, you can move all selenide properties to `selenide.properties` file and remove`setUp` method at all as selenide will automatically take properties from those file

Reply:
Good idea. Done.